Util/pool/build.gradle

```diff
@@ -0,0 +1,22 @@
+plugins {
+    id 'java-library'
+    id 'io.deephaven.project.register'
+}
+
+description 'util-pool: Optional extensions of the io.deephaven.base.pool module'
+
+dependencies {
+    api project(':Base')
+    api project(":IO")
+
+    compileOnly libs.jetbrains.annotations
+
+    testImplementation platform(libs.junit.bom)
+    testImplementation libs.junit.jupiter
+    testRuntimeOnly libs.junit.jupiter.engine
+    testRuntimeOnly libs.junit.platform.launcher
```

Comment:
You didn't move any test classes. So one wonders what you need these for, and whether there's been an oversight. I think there has.